### ISSUE TITLE
Missing aria-labels on points when chart displays a lot of points

### ISSUE THREAD

**Issue by kkesidis** [highcharts/highcharts#22580]
#### Expected behaviour

A Screen Reader (for example, VoiceOver) is able to read the information of each point (time, values)

#### Actual behaviour

Screen Readers (in my case VoiceOver) only read "graphics symbol"

#### Live demo with steps to reproduce
https://www.highcharts.com/demo/stock/stock-tools-gui

Select the "1Y" period and use a screen reader to navigate the chart. Please note, this is can be reproduced only when the chart displays a lot of points, so you'll likely need a decent size monitor and to close the "Code" part.

We can see that the points lack any `aria-label` and only have an `aria-hidden`.
![Image](https://github.com/user-attachments/assets/c7b48c81-b843-4c5a-b7ea-0db3f0aca4c5)

But changing to a smaller period (e.g. 1m), we can see that the points correctly have an `aria-label` and the screen reader properly reads their content
![Image](https://github.com/user-attachments/assets/0015af72-50e4-4190-a901-96720b48195a)

#### Product version
Highcharts JS v12.1.2

#### Affected browser(s)
Chrome, Firefox

**Comment by kkesidis** [comment]
Hello @marvin19 and thank you for the quick response. 

I completely missed this option.
